arch/dev/ext-flash/ext-flash.c

```diff
@@ -323,7 +323,7 @@ ext_flash_open(spi_device_t *conf)
   flash_spi_configuration = get_spi_conf(conf);
 
   /* Check if platform has ext-flash */
-  if(flash_spi_configuration->pin_spi_sck == GPIO_HAL_PIN_UNKNOWN) {
+  if(flash_spi_configuration->pin_spi_cs == GPIO_HAL_PIN_UNKNOWN) {
```

Comment:
I can leave it as SCK. Since the SPI pins are abstracted away by SPI controller IDs (SPI0, SPI1, etc.), the only SPI pin defined in the Board file for each SPI slave is the SPI CS. That means I have set all the others pins to PIN_UNKNOWN, as they are abstracted in the SPI controller. But, I can just expand the Board files to define all of the SPI pins, so this code snippet will work now.